### PR TITLE
[bitnami/airflow] Allow pod annotations for Airflow workers

### PR DESCRIPTION
**Pull request by pankajkoti**
**Description of the change**

Currently pod annotations cannot be given to Airflow workers. This is a PR to facilitate this use case

**Benefits**

We can specify additional pod annotations that custom Kubernetes clusters require. E.g. Specifying an AWS IAM role to be used by the pod

**Possible drawbacks**

**Applicable issues**

  - fixes https://github.com/bitnami/charts/issues/3513

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by pankajkoti** [comment]
looks like the test is failing in the pipeline as resources cannot connect to Postgres. Can someone please review this? Thank you!

File: bitnami/airflow/templates/statefulset-worker.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ spec:
     {{- end }}
   template:
     metadata:
+      annotations: {{- toYaml .Values.airflow.worker.podAnnotations | nindent 8 }}
```
Comment (miguelaeh) [inline bitnami/airflow/templates/statefulset-worker.yaml:32]: Please consider using the following:
```suggestion
      annotations: {{- include "airflow.tplValue" (dict "value" .Values.airflow.worker.podAnnotations "context" $) | nindent 8 }}
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!!
Please take a look at my comment.

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Comment by pankajkoti** [comment]
@miguelaeh can you please check if the Chart version is correct? And if yes, can you please also merge my request? I can use this chart in my code then. Thank you!

**Comment by miguelaeh** [comment]
Sure! I was waiting for the test to be passed.

**Comment by pankajkoti** [comment]
@miguelaeh Thanks for approving. Is it possible to make a release? or it would automatically in some time? Can I get some idea on how much time it would take approx. Thank you!

**Comment by pankajkoti** [comment]
I see bitnami/bot is creating releases. Hope it makes one soon :))

**Comment by miguelaeh** [comment]
Hi @pankajkoti ,
it is an automatic process. I can see the `6.4.0` version has been already released.
Regards.

**Pull request merged**